nighthawk/test/utility_test.cc

```diff
@@ -128,6 +129,20 @@ TEST_P(UtilityAddressResolutionTest, AddressResolutionBadAddresses) {
   EXPECT_THROW(testResolution("a..b", address_family), UriException);
 }
 
+TEST_P(UtilityAddressResolutionTest, resolveTwiceReturnsCached) {
```

Comment:
Same nit as in other PR about naming tests, starting with capital letter.

Reply:
Done